pkg/processor/k8sprocessor/kube/client.go

```diff
@@ -117,6 +119,51 @@ func New(
 	return c, err
 }
 
+// OwnerFeedbackLoop starts a loop that will receive notifications about updates
+// to endpoints objects which are then used to update corresponding pods with
+// ownership info.
+// For info it only updates the services attribute. See
+// Test_PodIsAnnotatedWithOwnershipInfoImmediatelyAfterOwnersAreCreated test in
+// client_test.go for details.
+func (c *WatchClient) OwnerFeedbackLoop(stopCh <-chan struct{}) {
```

Comment:
I wonder if we can piggyback on the infrastructure informers have for handling updates instead of reinventing it from scratch. Our owner cache is, after all, a kind of aggregate informer for multiple resource kinds.

Also, of note, is that while you don't need this for Services, if you want to do this for other owners, you need to track which pod is owned by which owner somewhere.

Reply:
I think I know what you mean but can you elaborate?

> you need to track which pod is owned by which owner somewhere.

That I know and this is what hold me back from implementing it (because of potential increased memory usage)